packages/nimble-components/src/theme-provider/design-token-comments.ts

```diff
@@ -293,5 +293,29 @@ export const comments: { readonly [key in TokenName]: string } = {
     mentionFontFamily: 'Font family for mention views',
     mentionFontSize: 'Font size for mention views',
     mentionFontWeight: 'Font weight for mention views',
-    mentionFontLineHeight: 'Font line height for mention views'
+    mentionFontLineHeight: 'Font line height for mention views',
+    calendarStaticEventBackgroundColor:
+        'Background color for static calendar events',
+    calendarDynamicEventBackgroundColor:
+        'Background color for dynamic calendar events',
+    calendarTransientEventBackgroundColor:
+        'Background color for transient calendar events',
+    calendarStaticEventBorderColor: 'Border color for static calendar events',
+    calendarTransientEventBorderColor:
+        'Border color for transient calendar events',
+    calendarStaticEventFontColor: 'Font color for static calendar events',
+    calendarDynamicEventFontColor: 'Font color for dynamic calendar events',
+    calendarTransientEventFontColor: 'Font color for transient calendar events',
+    calendarStaticEventHoverColor:
+        'Color while hovering static calendar events',
+    calendarDynamicEventHoverColor:
+        'Color while hovering dynamic calendar events',
+    calendarTransientEventHoverColor:
+        'Color while hovering transient calendar events',
+    calendarResourceSelectionBackgroundColor:
+        'Background color while calendar resource is selected',
+    calendarBlockedStrokeColor:
```

Comment:
is stroke meaningfully different than border? if not can we use the term border. if so can you explain / provide screenshots?

Reply:
These are strokes to represent the specific slot is busy/occupied based on the combined availability of the selected system with respect to its scheduled testplans.

![image](https://github.com/user-attachments/assets/de3f458c-fdd1-4bcd-b62d-8a76b42d352e)

Reply:
Given the fact that "stroke" is often used to mean "border", I'd vote to change this to something like "stripe". So `calendar-blocked-stripe-color` (open to improvements on that suggestion)

Reply:
I'd go for `calendar-fill-blocked-color` the specific fill color is independent from the fill shape / pattern (stripe today, could be rectangular, tomorrow).

Is there an argument that this is a calendar-event type? If so I'd alternatively say `calendar-event-fill-blocked-color`, but either of these options sound sane to me.

Reply:
I agree on `fill` rather than `stripe`. I'm less clear on the choice between the 3 variants. To me it's a bit different than an event and doesn't fill a whole row so I'd probably vote `calendar-fill-blocked-color`, but defer to @aagash-ni since you know the use case better.

Reply:
Updated it as calendarEventFillBlockedColor - `calendar-event-fill-blocked-color`